src/Controller/CustomerReorderAction.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\CustomerReorderPlugin\Controller;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+final class CustomerReorderAction
```

Comment:
I understand it's implemented for the grid item action? It just smells bad to have such an empty action :)

Reply:
Yes, it is and will be implemented in further PR :)